cg/cli/demultiplex/demux.py

```diff
@@ -121,66 +120,6 @@ def demultiplex_flow_cell(
         )
 
 
-@click.command(name="delete-flow-cell")
-@click.option(
-    "-f",
-    "--flow-cell-name",
-    required=True,
-    help="The name of the flow cell you want to delete, e.g. HVKJCDRXX",
-)
-@click.option(
-    "--demultiplexing-dir", is_flag=True, help="Delete flow cell demultiplexed dir on file system"
-)
-@click.option("--housekeeper", is_flag=True, help="Delete flow cell in housekeeper")
-@click.option("--init-files", is_flag=True, help="Delete flow cell init-files")
-@click.option("--run-dir", is_flag=True, help="Delete flow cell run on file system")
-@click.option(
-    "--sample-lane-sequencing-metrics",
-    is_flag=True,
-    help="Delete flow cell in sample lane sequencing metrics",
-)
-@click.option(
-    "--status-db",
-    is_flag=True,
-    help="Delete flow cell in status-db, if passed all other entries are also deleted",
-)
-@click.option("--yes", is_flag=True, help="Pass yes to click confirm")
-@DRY_RUN
-@click.pass_obj
-def delete_flow_cell(

```

Comment:
I'm not sure this is ever used ATM

Reply:
Shouldn't we leave this just in case? Maybe someone in prod uses it manually

Reply:
Can I create an issue for adding it?

Reply:
Question: would you want it to do the same as the automation in this PR, or even more?